third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -29,37 +29,47 @@ struct TestConfig {
     experiments: &'static [(&'static str, bool)],
     /// Run the tests with language version 1 (if true),
     /// or with latest language version (if false).
-    is_lang_v1: bool,
+    language_version: LanguageVersion,
     /// Path substrings for tests to include. If empty, all tests are included.
     include: &'static [&'static str],
     /// Path substrings for tests to exclude (applied after the include filter).
     /// If empty, no additional tests are excluded.
     exclude: &'static [&'static str],
 }
 
+/// Note that any config which has different output for a test directory
+/// *must* be added to the `SEPARATE_BASELINE` array below, so that a
+/// special output file `test.foo.exp` will be generated for the output
+/// of `test.move` for config `foo`.
 const TEST_CONFIGS: &[TestConfig] = &[
+    // Matches all default experiments
+    TestConfig {
+        name: "baseline",
+        runner: |p| run(p, get_config_by_name("baseline")),
+        experiments: &[],
+        language_version: LanguageVersion::V2_2,
```

Comment:
Whould we have a const like "LATEST_UNSTABLE_VERSION" for language and set it to V2_2 or it is just specific for v2.2?

Reply:
I really want a version like `LanguageVersion::LAMBDAS` but that seems like overkill.

Reply:
Oops, that one should not be `V2_2` but should be stable.  Hold on.

Reply:
fixed now.